### PR TITLE
[2.x] Fix displayed language name in admin panel

### PR DESCRIPTION
**Pull request by Ser1ous**
## Description

When add to language list pt-BR and pt he displayed in language list same. 
https://monosnap.com/file/fXBhpEkkcIOUgL45iHK8lbPn25FYPl
If change function from getDisplayLanguage to getDisplayName. Language list be more correct. 
https://monosnap.com/file/e2ohyGD3ZU0ZmvLsrHaFab2c5Aw2xM
It's work for all language modifie, like Chines Simply and Chines Tradition

## Related Issues

No issue, just find and fix :-)

### CODE REVIEW

**Comment by haringsrob** [comment]
Thank you @Ser1ous,

@ifox I do not see any issue with merging this in 2.8.0 but just checking as you may?

File: src/Helpers/i18n_helpers.php [hunk 1]
```diff
@@ -83,9 +83,9 @@ function getLanguageLabelFromLocaleCode($code, $native = false)
     {
         if (class_exists(Locale::class)) {
             if ($native) {
-                return ucfirst(Locale::getDisplayLanguage($code, $code));
+                return ucfirst(Locale::getDisplayName($code, $code));
```
Comment (haringsrob) [inline src/Helpers/i18n_helpers.php:86]: I had a quick check and we can drop ucfirst here as getDisplayName is already properly capitalized. 

Also, you can remove the } else { as it is returned early.

**Review by haringsrob — CHANGES_REQUESTED** [review]

File: src/Helpers/i18n_helpers.php (hunk 1 above)
Comment (haringsrob) [reply src/Helpers/i18n_helpers.php:86]: Same for the other call

Comment (Ser1ous) [reply src/Helpers/i18n_helpers.php]: I think need } else { in reason, when need translate for not current language

Comment (haringsrob) [reply src/Helpers/i18n_helpers.php]: Sorry, I should have been more clear as I did not mean the content of the else but rather to have it return properly.

```
if (true) {
   return 'bar';
} else {
   return 'foo';
}
```

is the same as:

```
if (true) {
   return 'bar';
}

return 'foo';
```

Comment (Ser1ous) [reply src/Helpers/i18n_helpers.php]: Ok, my fail :-( . Fixed this.

Comment (Ser1ous) [reply src/Helpers/i18n_helpers.php]: In this function has problem when class Locale not exists. But i check again on next weekend and if he be, then i fix too.

**Review by ifox — APPROVED** [review]

**Pull request merged**